packages/validator/src/slashingProtection/attestation/attestationByTargetRepository.ts

```diff
@@ -36,14 +39,16 @@ export class AttestationByTargetRepository {
     }); // casing doesn't matter
     this.bucketId = getBucketNameByValue(this.bucket);
     this.dbReqOpts = {bucketId: this.bucketId};
+    this.minKey = encodeKey(this.bucket, Buffer.alloc(0));
+    this.maxKey = encodeKey(this.bucket + 1, Buffer.alloc(0));
   }
 
   async getAll(pubkey: BLSPubkey, limit?: number): Promise<SlashingProtectionAttestation[]> {
     const attestations = await this.db.values({
-      ...this.dbReqOpts,
       limit,
       gte: this.encodeKey(pubkey, 0),
       lt: this.encodeKey(pubkey, Number.MAX_SAFE_INTEGER),
+      bucketId: this.bucketId,
```

Comment:
Unrelated change, just minor follow up to https://github.com/ChainSafe/lodestar/pull/5436

`FilterOptions` does not extend `DbReqOpts` and therefore is not guaranteed to be a superset of it, to avoid type issues in the future should rather not destructure `this.dbReqOpts` here

https://github.com/ChainSafe/lodestar/blob/3ed8d04c7971f753570f0d054a3c44bc2c382be1/packages/db/src/controller/interface.ts#L10

also inline with implementation here

https://github.com/ChainSafe/lodestar/blob/3ed8d04c7971f753570f0d054a3c44bc2c382be1/packages/db/src/abstractRepository.ts#L249-L252